examples/upbound-aws-provider/composite-resources/database-examples/aurora/aurora-postgresql.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: db.awsblueprint.io/v1alpha1
+kind: Aurora
+metadata:
+  name: test-aurora-postgresql-cluster
+  namespace: team-a
+spec:
+  writeConnectionSecretToRef:
```

Comment:
Use `PublishConnectionDetailsTo` instead. `writeConnectionSecretToRef` is planned to be deprecated.

Reply:
added new line.